Add push notification functionality to user profile

marimoslehi opened the pull request:
- Implemented push notification setup with expo-notifications
- Added notification permission handling in profile page
- Set up EAS project configuration for notifications
- Added notification button functionality in user profile

marimoslehi committed:
Add push notification functionality to user profile

- Implemented push notification setup with expo-notifications
- Added notification permission handling in profile page
- Set up EAS project configuration for notifications
- Added notification button functionality in user profile

ekjyotshinh committed:
Merge remote-tracking branch 'origin' into searchFunctionality

ekjyotshinh committed:
moved the notifications code from profile to the notifications screen

ekjyotshinh committed:
added a dummy remove device tokens functins to have the toggle work correctly

ekjyotshinh committed:
removed the testing code

ekjyotshinh reviewed (changes requested):
The code correctly extract the device details, and works well. Can you please add the logging in part (with expo account) to the readme so that others are not confused about it. Thanks a lot! Also please use a more descriptive/correct branch name that correctly aligns with the task.

marimoslehi committed:
Update Readme.md

Updated Readme.md :
-How to login to expo account
-How to set up Expo project ID so the notification works properly
- Update the .env file

ekjyotshinh committed:
Merge remote-tracking branch 'origin' into searchFunctionality

ekjyotshinh reviewed (approved):
Reviewed in the team meet
